webpack_new.config.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable */
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.ts',
+  target: 'web',
+  mode: 'production',
+
+  output: {
+    path: path.resolve(__dirname, 'browser_new'),
+    filename: 'index.js',
+    globalObject: '(typeof self !== \'undefined\' ? self : this)',
+    library: {
+      name: 'AsyncAPIParser',
+      type: 'umd'
```

Comment:
Is the lack of `export: 'default'` here making exporting just everything instead of the default export (which if I understand correctly is just `Parser`) ?

Reply:
Yes exactly

Reply:
Cool! Then, do we wanna expose every single method? Not sure. Then the package size will increment but not sure if this is really a critical aspect. Worth to check.

Reply:
I cannot figure out how to partially expose the functionality i.e. only parser and fromURL...

> Cool! Then, do we wanna expose every single method? Not sure. Then the package size will increment but not sure if this is really a critical aspect. Worth to check.

I don't care 😄 All I care about is that we expose the necessary functionality.

Reply:
If we want to minimize the size we should split the exposed libraries into individual packages instead of bundling everything.

Reply:
Fair enough. +1 in my side to go in that direction. Would then you move the `browser_new` content  stuff to the `browser` dir right?

Reply:
ping @jonaslagoni

Reply:
Soo, we just have a single folder called `browser` and make a breaking change? 🤔

Reply:
Why breaking change? 🤔

Reply:
Because you will no longer be able to use the same import statement i.e. `import Parser from '@asyncapi/parser/browser';` will be no more. 

Instead you have to do `import {Parser} from '@asyncapi/parser/browser';` if I recall correctly

Reply:
Ping @jonaslagoni

Reply:
Done